charts/lfx-v2-project-service/templates/httproute.yaml

```diff
@@ -0,0 +1,46 @@
+# Copyright The Linux Foundation and each contributor to LFX.
+# SPDX-License-Identifier: MIT
+---
+apiVersion: gateway.networking.k8s.io/v1
+kind: HTTPRoute
+metadata:
+  name: lfx-v2-project-service
+  namespace: {{ .Values.lfx.namespace }}
+spec:
+  parentRefs:
+  - name: {{ .Values.traefik.gateway.name }}
+    namespace: {{ .Values.traefik.gateway.namespace }}
+  hostnames:
+  - "lfx-api.{{ .Values.lfx.domain }}"
+  rules:
+  # Main application endpoints (with authentication)
+  - matches:
+    - path:
+        type: Exact
+        value: /projects
+    - path:
+        type: PathPrefix
+        value: /projects/
+    filters:
+    {{- if .Values.heimdall.enabled }}
+    - type: ExtensionRef
+      extensionRef:
+        group: traefik.io
+        kind: Middleware
+        name: heimdall
+        namespace: {{ .Values.traefik.gateway.namespace }}
+    {{- end }}
+    backendRefs:
+    - name: lfx-v2-project-service
+      port: {{ .Values.service.port }}
+  # Health check endpoints (no authentication required)
+  - matches:
+    - path:
+        type: Exact
+        value: /livez
+    - path:
+        type: Exact
+        value: /readyz
+    backendRefs:
+    - name: lfx-v2-project-service
+      port: {{ .Values.service.port }}
```

Comment:
These routes don't need to be exposed through Traefik as Kubernetes already has access to the through the service Cluster IP.